applications/connecting_applications_to_services/binding-workloads-using-sbo.adoc

```diff
@@ -42,6 +41,8 @@ include::modules/sbo-advanced-binding-options.adoc[leveloffset=+1]
 include::modules/sbo-binding-workloads-that-are-not-compliant-with-PodSpec.adoc[leveloffset=+1]
 include::modules/sbo-unbinding-workloads-from-a-backing-service.adoc[leveloffset=+1]
 
+[role="_additional-resources"]
```

Comment:
Please add an ID as well. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies

Reply:
Fixed